mlflow/tracking/client.py

```diff
@@ -1092,6 +1420,42 @@ def list_registered_models(
         :return: A PagedList of :py:class:`mlflow.entities.model_registry.RegisteredModel` objects
                  that can satisfy the search expressions. The pagination token for the next page
                  can be obtained via the ``token`` attribute of the object.
+
+        .. code-block:: python
+            :caption: Example
+
+            import mlflow
+            from mlflow.tracking import MlflowClient
+
+            def print_model_info(models):
+            for m in models:
+                print("--")
+                print("name: {}".format(m.name))
+                print("tags: {}".format(m.tags))
+                print("description: {}".format(m.description))
```

Comment:
```suggestion
            def print_model_info(models):
                for m in models:
                    print("--")
                    print("name: {}".format(m.name))
                    print("tags: {}".format(m.tags))
                    print("description: {}".format(m.description))
```

Reply:
ah, joys of cut-n-paste in the block code...:)